diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[

```

Comment:
docs should go in .rst files in the `docs/` subdirectory. You can copy one of the other files to get the format, and read more context on documentation standards here: https://docs.dfhack.org/en/latest/docs/dev/Documentation.html

Reply:
This is mostly about formatting. The parameters you have defined are fine.

Reply:
I'll see if I can figure that out.